toonz/sources/translations/german/toonzlib.ts

```diff
@@ -0,0 +1,610 @@
+    <message>
+        <source>Can&apos;t vectorize a level with no frames</source>
+        <translation>Kann keine Ebene ohne Frames vektorisieren</translation>
+    </message>
+    <message>
+        <source>Can&apos;t vectorize a %1 image</source>
+        <translation>Kann kein %1 Bild vektorisieren</translation>
+    </message>
+    <message>
+        <source>Bad argument (%1): should be an Image or a Level</source>
+        <translation>Inkorrektes Argument (%1): Es sollte ein Bild oder eine Ebene sein</translation>
+    </message>
+    <message>
+        <source>Invalid color : </source>
+        <translation>Ungültige Farbe : </translation>
+    </message>
+</context>
+<context>
+    <name>TScriptBinding::Rasterizer</name>
+    <message>
+        <source>Expected a vector image: %1</source>
+        <translation>Es ist kein Vektor-Bild: %1</translation>
+    </message>
+    <message>
+        <source>Expected a vector level: %1</source>
+        <translation>Es ist keine Vektor-Ebene: %1</translation>
+    </message>
+    <message>
+        <source>Argument must be a vector level or image : </source>
+        <translation>Argument muss eine Vektor-Ebene oder Bild sein : </translation>
+    </message>
+    <message>
+        <source>%1 has no palette</source>
+        <translation>%1 hat keine Palette</translation>
+    </message>
+</context>
+<context>
+    <name>TScriptBinding::Scene</name>
+    <message>
+        <source>File %1 doesn&apos;t exist</source>
+        <translation>Datei %1 existiert nicht</translation>
+    </message>
+    <message>
+        <source>Exception reading %1</source>
+        <translation>Ausnahme beim lesen von %1</translation>
+    </message>
+    <message>
+        <source>Exception writing %1</source>
+        <translation>Ausnahme beim schreiben von %1</translation>
+    </message>
+    <message>
+        <source>Bad level type (%1): must be Vector,Raster or ToonzRaster</source>
+        <translation>Inkorrekter Ebenentyp (%1): Es muss ein Vektor, Raster oder ToonzRaster sein</translation>
+    </message>
+    <message>
+        <source>Can&apos;t add the level: name(%1) is already used</source>
+        <translation>Kann die Ebene nicht hinzufügen: Der Name (%1) wird bereits verwendet</translation>
+    </message>
+    <message>
+        <source>Can&apos;t load this kind of file as a level : %1</source>
+        <translation>Kann diesen Dateityp nicht als Ebene laden : %1</translation>
+    </message>
+    <message>
+        <source>Could not load level %1</source>
+        <translation>Konnte Ebene %1 nicht laden</translation>
+    </message>
+    <message>
+        <source>Level is not included in the scene : %1</source>
+        <translation>Ebene ist nicht in der Szene vorhanden: %1</translation>
+    </message>
+    <message>
+        <source>%1 : Expected a Level instance or a level name</source>
+        <translation>%1 : Es ist keine Ebenen-Instanz oder ein Ebenen-Name</translation>
```

Comment:
Consider dropping the colon and "Es". Using es here is repetitive. Use weder ... noch to resolve the mismatch in grammatical gender.
"%1 ist weder eine Ebenen-Instanz noch ein Ebenen-Namen"